apps/crn-server/src/handlers/event/index-user-events-handler.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  AlgoliaSearchClient,
+  algoliaSearchClientFactory,
+} from '@asap-hub/algolia';
+import { EventResponse, ListResponse } from '@asap-hub/model';
+import { SquidexGraphql } from '@asap-hub/squidex';
+import { EventBridgeEvent } from 'aws-lambda';
+import { algoliaApiKey, algoliaAppId, algoliaIndex } from '../../config';
+import Events, { EventController } from '../../controllers/events';
+import logger from '../../utils/logger';
+import {
+  loopOverCustomCollection,
+  LoopOverCustomCollectionFetchOptions,
+} from '../../utils/loop-over-custom-colection';
+import { UserEvent, UserPayload } from '../event-bus';
+
+export const indexUserEventsHandler =
```

Comment:
All these handlers seem to be doing the same thing (with different filters).

Is there a way we can refactor to share the code across them?

Reply:
I do not think we want to do this